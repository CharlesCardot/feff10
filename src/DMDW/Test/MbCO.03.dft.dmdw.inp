  1
  6
 1 77.0
 0
MbCO.03.dft.dym
1
2 1 0     10.0
