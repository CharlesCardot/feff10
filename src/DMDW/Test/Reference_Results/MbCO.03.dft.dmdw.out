# Lanczos recursion order:    6
# Temperature:   77.00
# Dynamical matrix file: MbCO.03.dft.dym

--------------------------------------------------------------
 Path Indices:    1   2
 pDOS Centroid (THz):   27.580
 pDOS Poles:     Freq. (THz)    Weight
                  10.075       0.33565
                  28.541       0.45085
                  39.828       0.18253
                  47.422       0.03094
                  81.932       0.00000
                  98.861       0.00002
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.024   2.458
--------------------------------------------------------------
 Path Indices:    1   3
 pDOS Centroid (THz):   27.543
 pDOS Poles:     Freq. (THz)    Weight
                   9.911       0.33912
                  28.613       0.45123
                  40.036       0.18009
                  47.627       0.02954
                  91.137       0.00000
                  98.912       0.00002
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.025   2.493
--------------------------------------------------------------
 Path Indices:    1   4
 pDOS Centroid (THz):   27.547
 pDOS Poles:     Freq. (THz)    Weight
                  10.074       0.33721
                  28.543       0.45033
                  39.891       0.18286
                  47.439       0.02959
                  88.385       0.00000
                  98.866       0.00002
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.027   2.463
--------------------------------------------------------------
 Path Indices:    1   5
 pDOS Centroid (THz):   27.498
 pDOS Poles:     Freq. (THz)    Weight
                   9.863       0.33943
                  28.555       0.44822
                  39.842       0.18182
                  47.497       0.03051
                  80.280       0.00000
                  98.869       0.00002
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.026   2.503
--------------------------------------------------------------
 Path Indices:    1   6
 pDOS Centroid (THz):   31.315
 pDOS Poles:     Freq. (THz)    Weight
                  10.358       0.40001
                  28.111       0.14864
                  41.294       0.37640
                  48.695       0.07486
                  96.514       0.00001
                  98.767       0.00009
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.050   2.797
--------------------------------------------------------------
 Path Indices:    1   7
 pDOS Centroid (THz):   27.818
 pDOS Poles:     Freq. (THz)    Weight
                  10.211       0.41349
                  26.938       0.16996
                  37.474       0.38566
                  45.943       0.03081
                  95.303       0.00000
                  98.626       0.00008
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.099   2.962
--------------------------------------------------------------
 Path Indices:    1   8
 pDOS Centroid (THz):   31.345
 pDOS Poles:     Freq. (THz)    Weight
                  10.219       0.39980
                  28.212       0.14958
                  41.384       0.37648
                  48.759       0.07405
                  98.253       0.00007
                  99.404       0.00003
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.051   2.822
--------------------------------------------------------------
 Path Indices:    1   9
 pDOS Centroid (THz):   27.709
 pDOS Poles:     Freq. (THz)    Weight
                  10.115       0.41400
                  27.081       0.17563
                  37.422       0.38050
                  45.979       0.02980
                  95.640       0.00001
                  98.665       0.00007
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.098   2.986
--------------------------------------------------------------
 Path Indices:    1  10
 pDOS Centroid (THz):   31.281
 pDOS Poles:     Freq. (THz)    Weight
                  10.348       0.40109
                  28.132       0.14856
                  41.295       0.37656
                  48.718       0.07370
                  98.074       0.00005
                  99.156       0.00004
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.053   2.803
--------------------------------------------------------------
 Path Indices:    1  11
 pDOS Centroid (THz):   27.866
 pDOS Poles:     Freq. (THz)    Weight
                  10.191       0.41341
                  26.905       0.16710
                  37.495       0.38766
                  45.885       0.03176
                  95.656       0.00001
                  98.629       0.00007
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.101   2.964
--------------------------------------------------------------
 Path Indices:    1  12
 pDOS Centroid (THz):   31.329
 pDOS Poles:     Freq. (THz)    Weight
                  10.168       0.39924
                  28.041       0.14665
                  41.284       0.37887
                  48.712       0.07514
                  94.605       0.00000
                  98.688       0.00009
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.053   2.831
--------------------------------------------------------------
 Path Indices:    1  13
 pDOS Centroid (THz):   27.772
 pDOS Poles:     Freq. (THz)    Weight
                  10.110       0.41420
                  27.098       0.17246
                  37.487       0.38413
                  46.024       0.02914
                  95.247       0.00000
                  98.651       0.00008
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.098   2.985
--------------------------------------------------------------
 Path Indices:    1  14
 pDOS Centroid (THz):   34.302
 pDOS Poles:     Freq. (THz)    Weight
                  10.368       0.37424
                  24.443       0.42450
                  39.256       0.09891
                  49.983       0.03308
                  96.700       0.06920
                  98.288       0.00007
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.453   2.940
--------------------------------------------------------------
 Path Indices:    1  15
 pDOS Centroid (THz):   34.306
 pDOS Poles:     Freq. (THz)    Weight
                  10.340       0.37191
                  24.421       0.42647
                  39.157       0.09890
                  49.955       0.03347
                  96.701       0.06916
                  98.074       0.00009
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.454   2.939
--------------------------------------------------------------
 Path Indices:    1  16
 pDOS Centroid (THz):   34.304
 pDOS Poles:     Freq. (THz)    Weight
                  10.347       0.37411
                  24.440       0.42458
                  39.272       0.09882
                  50.034       0.03323
                  96.693       0.06918
                  98.219       0.00007
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.453   2.943
--------------------------------------------------------------
 Path Indices:    1  17
 pDOS Centroid (THz):   34.285
 pDOS Poles:     Freq. (THz)    Weight
                  10.315       0.37156
                  24.425       0.42727
                  39.153       0.09861
                  49.893       0.03336
                  96.677       0.06915
                  98.396       0.00005
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.455   2.943
--------------------------------------------------------------
 Path Indices:    1  18
 pDOS Centroid (THz):   34.104
 pDOS Poles:     Freq. (THz)    Weight
                  10.050       0.38373
                  27.198       0.35409
                  38.462       0.16044
                  44.582       0.04701
                  98.216       0.02690
                  98.879       0.02783
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.318   2.922
--------------------------------------------------------------
 Path Indices:    1  19
 pDOS Centroid (THz):   34.403
 pDOS Poles:     Freq. (THz)    Weight
                   9.972       0.37964
                  26.829       0.33582
                  38.764       0.21508
                  48.624       0.01279
                  98.218       0.02608
                  98.881       0.03059
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.338   2.922
--------------------------------------------------------------
 Path Indices:    1  20
 pDOS Centroid (THz):   34.056
 pDOS Poles:     Freq. (THz)    Weight
                   9.905       0.38487
                  27.259       0.35655
                  38.629       0.16188
                  44.761       0.04200
                  98.233       0.02827
                  98.900       0.02644
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.320   2.956
--------------------------------------------------------------
 Path Indices:    1  21
 pDOS Centroid (THz):   34.352
 pDOS Poles:     Freq. (THz)    Weight
                   9.848       0.38118
                  26.912       0.33703
                  38.760       0.21310
                  48.763       0.01212
                  98.232       0.02466
                  98.902       0.03191
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.339   2.952
--------------------------------------------------------------
 Path Indices:    1  22
 pDOS Centroid (THz):   34.108
 pDOS Poles:     Freq. (THz)    Weight
                  10.027       0.38284
                  27.187       0.35433
                  38.395       0.15914
                  44.492       0.04896
                  98.204       0.02671
                  98.867       0.02802
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.320   2.924
--------------------------------------------------------------
 Path Indices:    1  23
 pDOS Centroid (THz):   34.407
 pDOS Poles:     Freq. (THz)    Weight
                   9.951       0.37876
                  26.802       0.33608
                  38.757       0.21535
                  48.523       0.01313
                  98.206       0.02622
                  98.868       0.03045
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.340   2.924
--------------------------------------------------------------
 Path Indices:    1  24
 pDOS Centroid (THz):   34.085
 pDOS Poles:     Freq. (THz)    Weight
                   9.862       0.38393
                  27.250       0.35571
                  38.542       0.16040
                  44.658       0.04517
                  98.227       0.02860
                  98.893       0.02618
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.321   2.961
--------------------------------------------------------------
 Path Indices:    1  25
 pDOS Centroid (THz):   34.373
 pDOS Poles:     Freq. (THz)    Weight
                   9.827       0.38104
                  26.909       0.33603
                  38.785       0.21393
                  48.707       0.01242
                  98.227       0.02426
                  98.896       0.03231
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.340   2.955
--------------------------------------------------------------
 Path Indices:    1  26
 pDOS Centroid (THz):   92.022
 pDOS Poles:     Freq. (THz)    Weight
                  10.610       0.03903
                  24.883       0.04492
                  39.684       0.01107
                  50.565       0.00651
                  96.700       0.89794
                  98.492       0.00054
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.539   7.756
--------------------------------------------------------------
 Path Indices:    1  27
 pDOS Centroid (THz):   92.004
 pDOS Poles:     Freq. (THz)    Weight
                  10.531       0.03859
                  24.790       0.04548
                  39.389       0.01121
                  50.498       0.00664
                  96.701       0.89753
                  98.468       0.00054
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.540   7.766
--------------------------------------------------------------
 Path Indices:    1  28
 pDOS Centroid (THz):   92.019
 pDOS Poles:     Freq. (THz)    Weight
                  10.537       0.03873
                  24.786       0.04496
                  39.534       0.01118
                  50.567       0.00658
                  96.693       0.89800
                  98.468       0.00055
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.540   7.762
--------------------------------------------------------------
 Path Indices:    1  29
 pDOS Centroid (THz):   91.970
 pDOS Poles:     Freq. (THz)    Weight
                  10.545       0.03880
                  24.868       0.04570
                  39.503       0.01109
                  50.459       0.00655
                  96.677       0.89733
                  98.471       0.00052
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.541   7.772
--------------------------------------------------------------
 Path Indices:    1  30
 pDOS Centroid (THz):   86.514
 pDOS Poles:     Freq. (THz)    Weight
                  10.953       0.04914
                  30.126       0.10867
                  38.542       0.06316
                  47.629       0.03750
                  98.221       0.37892
                  98.883       0.36262
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.282   9.217
--------------------------------------------------------------
 Path Indices:    1  31
 pDOS Centroid (THz):   87.235
 pDOS Poles:     Freq. (THz)    Weight
                  10.844       0.04874
                  29.324       0.09162
                  38.117       0.07062
                  47.313       0.03248
                  98.226       0.36611
                  98.886       0.39042
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.315   9.110
--------------------------------------------------------------
 Path Indices:    1  32
 pDOS Centroid (THz):   86.519
 pDOS Poles:     Freq. (THz)    Weight
                  10.821       0.04907
                  30.138       0.10901
                  38.557       0.06284
                  47.728       0.03749
                  98.237       0.39727
                  98.903       0.34432
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.284   9.242
--------------------------------------------------------------
 Path Indices:    1  33
 pDOS Centroid (THz):   87.234
 pDOS Poles:     Freq. (THz)    Weight
                  10.745       0.04883
                  29.401       0.09268
                  38.168       0.07011
                  47.427       0.03251
                  98.241       0.34719
                  98.907       0.40867
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.315   9.136
--------------------------------------------------------------
 Path Indices:    1  34
 pDOS Centroid (THz):   86.501
 pDOS Poles:     Freq. (THz)    Weight
                  10.931       0.04910
                  30.114       0.10879
                  38.547       0.06322
                  47.593       0.03740
                  98.210       0.37647
                  98.870       0.36502
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.283   9.222
--------------------------------------------------------------
 Path Indices:    1  35
 pDOS Centroid (THz):   87.222
 pDOS Poles:     Freq. (THz)    Weight
                  10.791       0.04848
                  29.180       0.08999
                  37.962       0.07215
                  47.238       0.03283
                  98.214       0.36814
                  98.874       0.38841
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.316   9.117
--------------------------------------------------------------
 Path Indices:    1  36
 pDOS Centroid (THz):   86.537
 pDOS Poles:     Freq. (THz)    Weight
                  10.782       0.04895
                  30.138       0.10894
                  38.570       0.06260
                  47.670       0.03735
                  98.232       0.40231
                  98.897       0.33985
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.285   9.242
--------------------------------------------------------------
 Path Indices:    1  37
 pDOS Centroid (THz):   87.230
 pDOS Poles:     Freq. (THz)    Weight
                  10.707       0.04869
                  29.327       0.09136
                  38.064       0.07135
                  47.344       0.03278
                  98.236       0.34208
                  98.901       0.41374
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.315   9.141
--------------------------------------------------------------
 Path Indices:    1  38
 pDOS Centroid (THz):   28.951
 pDOS Poles:     Freq. (THz)    Weight
                   8.622       0.28713
                  28.227       0.33800
                  35.244       0.28060
                  45.868       0.09420
                 100.464       0.00007
                 109.641       0.00001
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.049   2.509
--------------------------------------------------------------
 Path Indices:    1  39
 pDOS Centroid (THz):   32.548
 pDOS Poles:     Freq. (THz)    Weight
                   8.184       0.41337
                  26.555       0.07815
                  39.176       0.14450
                  45.419       0.36349
                 100.238       0.00046
                 109.633       0.00003
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.056   3.364
--------------------------------------------------------------
 Path Indices:    1  40
 pDOS Centroid (THz):   30.785
 pDOS Poles:     Freq. (THz)    Weight
                   8.314       0.43423
                  28.673       0.12728
                  39.739       0.24985
                  46.976       0.18837
                  99.982       0.00027
                 109.560       0.00001
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.099   3.456
--------------------------------------------------------------
 Path Indices:    1  41
 pDOS Centroid (THz):   35.714
 pDOS Poles:     Freq. (THz)    Weight
                   8.503       0.39365
                  28.172       0.30580
                  38.514       0.15574
                  46.097       0.09776
                  99.875       0.00004
                 109.650       0.04701
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.224   2.898
--------------------------------------------------------------
 Path Indices:    1  42
 pDOS Centroid (THz):   38.964
 pDOS Poles:     Freq. (THz)    Weight
                   9.156       0.37134
                  30.254       0.32225
                  46.547       0.23876
                  86.663       0.00012
                  99.875       0.06738
                 107.609       0.00016
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.267   2.930
--------------------------------------------------------------
 Path Indices:    1  43
 pDOS Centroid (THz):   43.898
 pDOS Poles:     Freq. (THz)    Weight
                  11.532       0.19420
                  14.349       0.31214
                  35.091       0.00025
                  61.011       0.49342
                  97.101       0.00000
                 109.619       0.00000
 Path Length (Ang), s^2 (1e-3 Ang^2):  1.828   2.390
--------------------------------------------------------------
 Path Indices:    1  44
 pDOS Centroid (THz):   35.807
 pDOS Poles:     Freq. (THz)    Weight
                  11.053       0.19790
                  14.111       0.49026
                  35.915       0.00019
                  61.011       0.31165
                  98.226       0.00000
                 109.591       0.00000
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.993   2.349
--------------------------------------------------------------
 Path Indices:    1  45
 pDOS Centroid (THz):   46.105
 pDOS Poles:     Freq. (THz)    Weight
                   9.134       0.07951
                  34.489       0.59418
                  44.031       0.23033
                  61.588       0.00046
                 100.163       0.05741
                 100.900       0.03812
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.293  16.428
--------------------------------------------------------------
 Path Indices:    1  46
 pDOS Centroid (THz):   49.822
 pDOS Poles:     Freq. (THz)    Weight
                   9.370       0.08086
                  35.071       0.53811
                  43.937       0.24827
                  71.335       0.00008
                 100.252       0.13262
                 108.799       0.00006
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.315  15.825
--------------------------------------------------------------
 Path Indices:    1  47
 pDOS Centroid (THz):   97.741
 pDOS Poles:     Freq. (THz)    Weight
                   9.332       0.04174
                  32.837       0.09783
                  43.300       0.08566
                  51.151       0.00316
                 100.220       0.00089
                 109.650       0.77072
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.140   8.452
--------------------------------------------------------------
 Path Indices:    1  48
 pDOS Centroid (THz):   90.851
 pDOS Poles:     Freq. (THz)    Weight
                   9.792       0.04330
                  32.394       0.10623
                  42.538       0.04146
                  99.608       0.58595
                 100.410       0.22273
                 109.404       0.00032
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.282   8.576
